packages/circuit-ui/components/Selector/Selector.tsx

```diff
@@ -147,20 +158,37 @@ const inputStyles = ({ theme }: StyleProps) => css`
   label: selector__input;
   ${hideVisually()};
 
-  &:focus + label::before {
+  &:focus + label {
     ${focusOutline({ theme })};
   }
 
   &:checked + label {
+    position: relative;
     background-color: ${theme.colors.p100};
+    border-color: transparent;
 
+    // Servers only as a container for the border,
```

Comment:
Thanks a lot for adding a comment on what this does! 

Is there a reason we didn't keep the previous implementation? I see this one resorts to `calc()` which might be slightly less performant and looks a bit less clean IMO. But is it doing anything new?

```suggestion
    // Serves only as a container for the border,
```

Reply:
Yeah, this one is tricky to explain with text, please see the video https://cln.sh/DpVKQx

Reply:
I refactored this part, because I realized that `Input` is not actually using `border`, it mimics it with `box-shadow`, meaning `Selector` and `Input` would still be misaligned when using my previous implementation 🤦‍♂️

What I do now:
* Using `box-shadow` to mimic border, just as the input does
* Using `::before` only as a container for the focus outline (still need to align it with the label's border using `calc()` so that focus outline does not cover label's border)

Let me know if this is not clear, we can have a chat and I'll try to explain things more visually.

Reply:
The reason why the outline was applied with a pseudo-element was to prevent the selector height from changing when it's checked (and the border increases to 2px).